content/doc/administration/requirements/java.adoc

```diff
@@ -5,29 +5,26 @@ title:  Java requirements
 
 There are separate run and job execution requirements for Jenkins installations.
 
-## Running Jenkins
```

Comment:
Why the removal?

Reply:
didn't mean to remove that, must have been pasted over. It's back now, sorry about that!